app/src/main/java/org/p2p/wallet/striga/wallet/models/StrigaInitiateOnchainWithdrawalDetails.kt

```diff
@@ -0,0 +1,50 @@
+package org.p2p.wallet.striga.wallet.models
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import org.threeten.bp.ZonedDateTime
+import java.math.BigDecimal
+import java.math.BigInteger
+import kotlinx.parcelize.Parcelize
+import org.p2p.wallet.striga.wallet.models.ids.StrigaAccountId
+import org.p2p.wallet.striga.wallet.models.ids.StrigaWalletId
+import org.p2p.wallet.striga.wallet.models.ids.StrigaWithdrawalChallengeId
+
+@Parcelize
+data class StrigaInitiateOnchainWithdrawalDetails(
+    val challengeId: StrigaWithdrawalChallengeId,
+    val dateExpires: ZonedDateTime,
+    val transaction: Transaction,
+    val feeEstimate: FeeEstimate,
+) : Parcelable {
+
+    /**
+     * @param userId is the user id in Striga
+     * @param amount in wei/satoshi or cents, depending on is the currency a fiat/stable or plain crypto
+     */
+    @Parcelize
+    data class Transaction(
```

Comment:
I will rename it to `WithdrawalTransactionDetails` to avoid confusions as It functions as an off-ramp for striga and an on-ramp for us, I think it's better to keep the `transaction`